core/src/main/java/io/questdb/cutlass/pgwire/CleartextPasswordPgWireAuthenticator.java

```diff
@@ -383,7 +384,7 @@ private int processPasswordMessage() throws BadProtocolException {
 
         long hi = PGConnectionContext.getStringLength(recvBufReadPos, msgLimit, "bad password length");
         dbcs.of(recvBufReadPos, hi);
```

Comment:
Is `dbcs.of(recvBufReadPos, hi);` still needed here ?

Reply:
good one, thanks!